README.md

```diff
@@ -18,57 +20,42 @@ To hash a list of bytes, invoke the [`convert`][convert] method on the
 objects.
 
 ```dart
-import 'package:crypto/crypto.dart';
 import 'dart:convert'; // for the UTF8.encode method
+import 'package:crypto/crypto.dart';
 
 void main() {
-  var bytes = UTF8.encode("foobar"); // data being hashed
+  var bytes = UTF8.encode('foobar'); // data being hashed
 
   var digest = sha1.convert(bytes);
 
-  print("Digest as bytes: ${digest.bytes}");
-  print("Digest as hex string: $digest");
+  print('Digest as bytes: ${digest.bytes}');
+  print('Digest as hex string: $digest');
 }
 ```
 
-### Digest on chunked input
+### Digest Stream input
 
-If the input data is not available as a _single_ list of bytes, use
-the chunked conversion approach.
+If the input data is a `Stream<List<int>>`:
 
-Invoke the [`startChunkedConversion`][startChunkedConversion] method
-to create a sink for the input data. On the sink, invoke the `add`
-method for each chunk of input data, and invoke the `close` method
-when all the chunks have been added. The digest can then be retrieved
-from the `Sink<Digest>` used to create the input data sink.
 
 ```dart
+import 'dart:async';
 import 'dart:convert';
 import 'package:crypto/crypto.dart';
-import 'package:crypto/src/digest_sink.dart';
 
-void main() {
-  var firstChunk = UTF8.encode("foo");
-  var secondChunk = UTF8.encode("bar");
-
-  var ds = new DigestSink();
-  var s = sha1.startChunkedConversion(ds);
-  s.add(firstChunk);
-  s.add(secondChunk); // call `add` for every chunk of input data
-  s.close();
-  var digest = ds.value;
-
-  print("Digest as bytes: ${digest.bytes}");
-  print("Digest as hex string: $digest");
+main() async {
+  var byteStream =
+      new Stream<String>.fromIterable(['foo', 'bar']).transform(UTF8.encoder);
+
+  var digest = await byteStream
+      .transform(sha1) // Convert to a Stream<Digest>
+      .single; // There is only one
```

Comment:
We should just add `Future<Digest> Hash.convertStream(Stream<List<int>> stream)`.

Reply:
Zero argument here.

Reply:
Can you add this? It shouldn't be a very big change.

Reply:
Can you add this? It shouldn't be a very big change.